internal_ws/ykcompile/src/lib.rs

```diff
@@ -130,26 +129,6 @@ pub struct CompiledTrace {
 }
 
 impl CompiledTrace {
-    /// Execute the trace by calling (not jumping to) the first instruction's address. Returns a
```

Comment:
I'm very surprised by the removal of this hunk. Investigating.

Reply:
I see, this code was copied into `ykshim_client` and the old version wasn't removed.